contribute.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Contribute
+order: 40
+---
+
+# Contribute to mozregression
+
+Contributors are welcome on mozregression!
```

Comment:
I think I'd prefer to keep this information on the wiki, in keeping with how we handle other projects in auto-tools. Maybe just use the existing verbiage on mozilla.github.io/mozregression:

See the mozregression project page for more information on how to contribute.